recipes/r-lasr/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "lasr" %}
+{% set version = "0.13.5" %}
+{% set posix = "m2-" if win else "" %}
+{% set native = "m2w64-" if win else "" %}
+
+package:
+  name: r-{{ name }}
+  version: '{{ version }}'
+
+source:
+  url: https://r-lidar.r-universe.dev/src/contrib/lasR_{{ version }}.tar.gz
+  sha256: c3a60e35dcb2e6403b40505928ccb22733681d63ce4926272c3c85eaa6e38e8a
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+
+requirements:
+  build:
+    - cross-r-base ${{ r_base }}  # [build_platform != target_platform]
+    - r-rcpp  # [build_platform != target_platform]
+    - "{{ compiler('c') }}"  # [unix]
+    - "{{ stdlib('c') }}"  # [not win]
+    - "{{ compiler('cxx') }}"  # [unix]
+    - {{ posix }}autoconf
+    - {{ posix }}automake  # [not win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - posix  # [win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+
+  
+  host:
+    - r-base
+    - r-rcpp
+    - r-r.utils
+    - libgdal-core
+    - libgomp
+    - proj
+    - geos
+    - _openmp_mutex
+
+  run:
+    - r-base
+    - libgdal-core
+    - proj
+    - geos
+
+test:
+  commands:
+    - $R -e "library('lasR')"  # [not win]
+    - "\"%R%\" -e \"library('lasR')\""  # [win]
+
+about:
+  home: https://github.com/r-lidar/lasR
+  summary: Fast and pipeable airborne lidar processing tools. Read/write 'las' and 'laz' files, computation of metrics in area based approach, point filtering, normalization, individual tree segmentation and other manipulations in a powerful and versatile processing chain.
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - wildintellect
```

We would prefer that `conda-forge/r` always be include on R recipes:

```suggestion
    - conda-forge/r
    - wildintellect
```